docs/content/documentation/templates/overview.md

```diff
@@ -212,10 +212,20 @@ As a security precaution, if this file is outside the main site directory, your
 {% set data = load_data(path="content/blog/story/data.toml") %}
 ```
 
+The optional `required` boolean argument can be set to false so that missing data does not produce an error, but returns a null value instead. For URLs, all HTTP errors are discarded. For local paths, missing file errors are discarded, but permissions errors are kept. In both cases, invalid data that could not be parsed to the requested data format will still produce an error.
```

Comment:
It's not clear to me what you mean by discarded. 

How about:

The optional `required` boolean argument can be set to `false` so that missing data (HTTP error or local file not found) does not produce an error, but returns a null value instead. However, permission issues with a local file and  invalid data that could not be parsed to the requested data format will still produce an error even with `request=false`.

?